plsql/ExcelGen.pkb

```diff
@@ -62,6 +62,7 @@ create or replace package body ExcelGen is
   
   DEFAULT_DATE_FMT       constant varchar2(32) := 'dd/mm/yyyy hh:mm:ss';
   DEFAULT_TIMESTAMP_FMT  constant varchar2(32) := 'dd/mm/yyyy hh:mm:ss.000';
+  DEFAULT_NUM_FMT        constant varchar2(32) := '0.000000';
```

Comment:
For backward compatibility, DEFAULT_NUM_FMT should be left empty, meaning the "General" cell format will apply if nothing else specified via setNumFmt.